lib/lib/day_night_timepicker_android.dart

```diff
@@ -45,72 +45,148 @@ class DayNightTimePickerAndroidState extends State<DayNightTimePickerAndroid> {
         ? timeState.time.hour
         : timeState.time.hourOfPeriod;
 
-    return FilterWrapper(
-      child: WrapperDialog(
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: <Widget>[
-            DayNightBanner(),
-            WrapperContainer(
+    Orientation currentOrientation = MediaQuery.of(context).orientation;
+
+    return currentOrientation == Orientation.portrait
+        ? FilterWrapper(
```

Comment:
I think we can always wrap with `SingleChildScrollView` and then pass `AlwaysScrollableScrollPhysics()` or `NeverScrollableScrollPhysics()` based on the orientation.